minimal.js

```diff
@@ -128,20 +141,14 @@ module.exports = function (dirname, keys, opts) {
       return cb(err)
     }
 
-    queue(msg, function (err) {
+    queue(msg, function (err, message) {
       if (err) return cb(err)
-      var data = state.queue[state.queue.length - 1]
-      flush.add(() => cb(null, data))
+      flush.add(() => cb(null, message))
     })
   }))
 
-  db.buffer = function buffer () {
```

Comment:
Removed because it was a shortcut method to introspect Async-Write and wasn't exported or used anywhere.